src/main/java/dk/kb/present/copyright/XsltCopyrightMapper.java

```diff
@@ -67,6 +72,13 @@ public static HashMap<String,String> xsltCopyrightTransformer(String modMedsXML)
         if (copyrightAccessDto.getImageUrl() != null) {
           solrFieldsMap.put("imageurl",mapper.getImageUrl());
         }
+        }
+        catch(Exception e) {
```

Comment:
Line 66 says it should always be there, but it is not! (metadata error).
Code breaks... And for anything to work it needs a default value, and that is what happens here. 
It will be fixed when the meta data format has been corrected.